tests/e2e/test_slippage_investigation.py

```diff
@@ -16,7 +16,7 @@ def test_no_outrageous_slippage(self):
         which tx are having high slippage values in dollar terms
         """
         dune = DuneAPI.new_from_environment()
-        period = AccountingPeriod("2022-03-10", 1)
+        period = AccountingPeriod("2022-06-08", 1)
```

Comment:
@josojo - I just realized that this date is a Wednesday (not the probably intended Tuesday "2022-06-07"). This isn't a huge problem, but we should fix it.